src/components/new-safe/CreateSafe/index.tsx

```diff
@@ -22,24 +24,56 @@ export type NewSafeFormData = {
   mobileOwners: NamedAddress[]
 }
 
-export const CreateSafeSteps: TxStepperProps<NewSafeFormData>['steps'] = [
-  {
-    title: 'Select network and name Safe',
-    subtitle: 'Select the network on which to create your Safe',
-    render: (data, onSubmit, onBack) => <CreateSafeStep1 onSubmit={onSubmit} onBack={onBack} data={data} />,
+const staticHints: Record<number, { title: string; variant: AlertColor; steps: { title: string; text: string }[] }> = {
+  0: {
+    title: 'Safe Creation',
+    variant: 'info',
+    steps: [
+      {
+        title: 'Network fee',
+        text: 'Deploying your Safe contract requires the payment of the associated network fee with your connected wallet. An estmation will be provided in the last step.',
+      },
+    ],
   },
-  {
-    title: 'Owners and confirmations',
-    subtitle:
-      'Here you can add owners to your Safe and determine how many owners need to confirm before making a successful transaction',
-    render: (data, onSubmit, onBack) => <CreateSafeStep2 onSubmit={onSubmit} onBack={onBack} data={data} />,
+  1: {
+    title: 'Safe Creation',
+    variant: 'info',
+    steps: [
+      {
+        title: 'Flat hierarchy',
+        text: 'Every owner has the same rights within the Safe and can propose, sign and execute transactions.',
+      },
+      {
+        title: 'Managing Owners',
+        text: 'You can always change the amount of owners and required confirmations in your Safe at a later stage after the creation.',
+      },
+      {
+        title: 'Safe Setup',
+        text: 'Not sure how many owners and confirmations you need for your Safe? Learn more about setting up your Safe.',
+      },
+    ],
   },
-  {
-    title: 'Review',
-    subtitle: `You're about to create a new Safe and will have to confirm a transaction with your currently connected wallet.`,
-    render: (data, onSubmit, onBack) => <CreateSafeStep3 onSubmit={onSubmit} onBack={onBack} data={data} />,
+  2: {
+    title: 'Safe Creation',
+    variant: 'info',
+    steps: [
+      {
+        title: 'Wait for the creation',
+        text: 'Depending on the network congestion, it can take some time until the transaction is successfully included on the network and picked up by our services.',
```

Comment:
```suggestion
        text: 'Depending on network congestion, it can take some time until the transaction is successfully added to the network and picked up by our services.',
```